### ISSUE TITLE
Module Installation problem

### ISSUE THREAD

**Issue by shadrackkibet** [spyder-ide/spyder#5807]
Problem installing spy2 module using pip.

I run this 

> !pip install rpy2

 in ipython console

This is the error i got.
`Collecting rpy2
  Using cached rpy2-2.9.1.tar.gz
Requirement already satisfied: six in c:\users\shadrack\anaconda3\lib\site-packages (from rpy2)
Requirement already satisfied: jinja2 in c:\users\shadrack\anaconda3\lib\site-packages (from rpy2)
Requirement already satisfied: MarkupSafe>=0.23 in c:\users\shadrack\anaconda3\lib\site-packages (from jinja2->rpy2)
Building wheels for collected packages: rpy2
  Running setup.py bdist_wheel for rpy2: started
  Running setup.py bdist_wheel for rpy2: finished with status 'error'
  Complete output from command C:\Users\SHADRACK\Anaconda3\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\SHADRACK\\AppData\\Local\\Temp\\pip-build-ohwpm7w_\\rpy2\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d C:\Users\SHADRACK\AppData\Local\Temp\tmpfglffk9ppip-wheel- --python-tag cp36:
  R version 3.4.2 (2017-09-28) -- "Short Summer"
  Copyright (C) 2017 The R Foundation for Statistical Computing
  Platform: x86_64-w64-mingw32/x64 (64-bit)
  
  R is free software and comes with ABSOLUTELY NO WARRANTY.
  You are welcome to redistribute it under the terms of the
  GNU General Public License versions 2 or 3.
  For more information about these matters see
  http://www.gnu.org/licenses/.
  
  R version 3.4.2 (2017-09-28) -- "Short Summer"
  C:\Users\SHADRACK\Anaconda3\R\bin\x64\R CMD config --ldflags
  bash.exe: warning: could not find /tmp, please create!
  C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 177: make: command not found
  C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 178: make: command not found
  C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 179: make: command not found
  C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 180: make: command not found
  C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 181: make: command not found
  R was not built as a library
  C:\Users\SHADRACK\Anaconda3\R\bin\x64\R CMD config --cppflags
  bash.exe: warning: could not find /tmp, please create!
  C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 177: make: command not found
  C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 178: make: command not found
  C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 179: make: command not found
  C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 180: make: command not found
  C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 181: make: command not found
  R was not built as a library
  C:\Users\SHADRACK\AppData\Local\Temp\pip-build-ohwpm7w_\rpy2\setup.py:282: UserWarning: No include specified
    warnings.warn('No include specified')
  C:\Users\SHADRACK\AppData\Local\Temp\pip-build-ohwpm7w_\rpy2\setup.py:293: UserWarning: No libraries as -l arguments to the compiler.
    warnings.warn('No libraries as -l arguments to the compiler.')
  
      Compilation parameters for rpy2's C components:
          include_dirs    = []
          library_dirs    = []
          libraries       = []
          extra_link_args = ['-m64']
  
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build\lib.win-amd64-3.6
  creating build\lib.win-amd64-3.6\rpy2
  copying .\rpy\rpy_classic.py -> build\lib.win-amd64-3.6\rpy2
  copying .\rpy\situation.py -> build\lib.win-amd64-3.6\rpy2
  copying .\rpy\tests.py -> build\lib.win-amd64-3.6\rpy2
  copying .\rpy\tests_rpy_classic.py -> build\lib.win-amd64-3.6\rpy2
  copying .\rpy\__init__.py -> build\lib.win-amd64-3.6\rpy2
  creating build\lib.win-amd64-3.6\rpy2\rlike
  copying .\rpy\rlike\container.py -> build\lib.win-amd64-3.6\rpy2\rlike
  copying .\rpy\rlike\functional.py -> build\lib.win-amd64-3.6\rpy2\rlike
  copying .\rpy\rlike\indexing.py -> build\lib.win-amd64-3.6\rpy2\rlike
  copying .\rpy\rlike\__init__.py -> build\lib.win-amd64-3.6\rpy2\rlike
  creating build\lib.win-amd64-3.6\rpy2\rlike\tests
  copying .\rpy\rlike\tests\test_container.py -> build\lib.win-amd64-3.6\rpy2\rlike\tests
  copying .\rpy\rlike\tests\test_functional.py -> build\lib.win-amd64-3.6\rpy2\rlike\tests
  copying .\rpy\rlike\tests\test_indexing.py -> build\lib.win-amd64-3.6\rpy2\rlike\tests
  copying .\rpy\rlike\tests\__init__.py -> build\lib.win-amd64-3.6\rpy2\rlike\tests
  creating build\lib.win-amd64-3.6\rpy2\rinterface
  copying .\rpy\rinterface\__init__.py -> build\lib.win-amd64-3.6\rpy2\rinterface
  creating build\lib.win-amd64-3.6\rpy2\rinterface\tests
  copying .\rpy\rinterface\tests\test_Device.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
  copying .\rpy\rinterface\tests\test_EmbeddedR.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
  copying .\rpy\rinterface\tests\test_Sexp.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
  copying .\rpy\rinterface\tests\test_SexpClosure.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
  copying .\rpy\rinterface\tests\test_SexpEnvironment.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
  copying .\rpy\rinterface\tests\test_SexpExtPtr.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
  copying .\rpy\rinterface\tests\test_SexpSymbol.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
  copying .\rpy\rinterface\tests\test_SexpVector.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
  copying .\rpy\rinterface\tests\test_SexpVectorNumeric.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
  copying .\rpy\rinterface\tests\__init__.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
  creating build\lib.win-amd64-3.6\rpy2\robjects
  copying .\rpy\robjects\constants.py -> build\lib.win-amd64-3.6\rpy2\robjects
  copying .\rpy\robjects\conversion.py -> build\lib.win-amd64-3.6\rpy2\robjects
  copying .\rpy\robjects\environments.py -> build\lib.win-amd64-3.6\rpy2\robjects
  copying .\rpy\robjects\functions.py -> build\lib.win-amd64-3.6\rpy2\robjects
  copying .\rpy\robjects\help.py -> build\lib.win-amd64-3.6\rpy2\robjects
  copying .\rpy\robjects\language.py -> build\lib.win-amd64-3.6\rpy2\robjects
  copying .\rpy\robjects\methods.py -> build\lib.win-amd64-3.6\rpy2\robjects
  copying .\rpy\robjects\numpy2ri.py -> build\lib.win-amd64-3.6\rpy2\robjects
  copying .\rpy\robjects\packages.py -> build\lib.win-amd64-3.6\rpy2\robjects
  copying .\rpy\robjects\packages_utils.py -> build\lib.win-amd64-3.6\rpy2\robjects
  copying .\rpy\robjects\pandas2ri.py -> build\lib.win-amd64-3.6\rpy2\robjects
  copying .\rpy\robjects\robject.py -> build\lib.win-amd64-3.6\rpy2\robjects
  copying .\rpy\robjects\vectors.py -> build\lib.win-amd64-3.6\rpy2\robjects
  copying .\rpy\robjects\__init__.py -> build\lib.win-amd64-3.6\rpy2\robjects
  creating build\lib.win-amd64-3.6\rpy2\robjects\tests
  copying .\rpy\robjects\tests\testArray.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
  copying .\rpy\robjects\tests\testDataFrame.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
  copying .\rpy\robjects\tests\testEnvironment.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
  copying .\rpy\robjects\tests\testFormula.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
  copying .\rpy\robjects\tests\testFunction.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
  copying .\rpy\robjects\tests\testHelp.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
  copying .\rpy\robjects\tests\testLanguage.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
  copying .\rpy\robjects\tests\testMethods.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
  copying .\rpy\robjects\tests\testNumpyConversions.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
  copying .\rpy\robjects\tests\testPackages.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
  copying .\rpy\robjects\tests\testPandasConversions.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
  copying .\rpy\robjects\tests\testRObject.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
  copying .\rpy\robjects\tests\testRobjects.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
  copying .\rpy\robjects\tests\testVector.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
  copying .\rpy\robjects\tests\__init__.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
  creating build\lib.win-amd64-3.6\rpy2\robjects\lib
  copying .\rpy\robjects\lib\dplyr.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib
  copying .\rpy\robjects\lib\ggplot2.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib
  copying .\rpy\robjects\lib\grdevices.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib
  copying .\rpy\robjects\lib\grid.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib
  copying .\rpy\robjects\lib\test_dplyr.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib
  copying .\rpy\robjects\lib\tidyr.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib
  copying .\rpy\robjects\lib\__init__.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib
  creating build\lib.win-amd64-3.6\rpy2\robjects\lib\tests
  copying .\rpy\robjects\lib\tests\test_dplyr.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib\tests
  copying .\rpy\robjects\lib\tests\test_ggplot2.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib\tests
  copying .\rpy\robjects\lib\tests\test_grdevices.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib\tests
  copying .\rpy\robjects\lib\tests\__init__.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib\tests
  creating build\lib.win-amd64-3.6\rpy2\interactive
  copying .\rpy\interactive\packages.py -> build\lib.win-amd64-3.6\rpy2\interactive
  copying .\rpy\interactive\process_revents.py -> build\lib.win-amd64-3.6\rpy2\interactive
  copying .\rpy\interactive\__init__.py -> build\lib.win-amd64-3.6\rpy2\interactive
  creating build\lib.win-amd64-3.6\rpy2\interactive\tests
  copying .\rpy\interactive\tests\__init__.py -> build\lib.win-amd64-3.6\rpy2\interactive\tests
  creating build\lib.win-amd64-3.6\rpy2\ipython
  copying .\rpy\ipython\ggplot.py -> build\lib.win-amd64-3.6\rpy2\ipython
  copying .\rpy\ipython\html.py -> build\lib.win-amd64-3.6\rpy2\ipython
  copying .\rpy\ipython\rmagic.py -> build\lib.win-amd64-3.6\rpy2\ipython
  copying .\rpy\ipython\__init__.py -> build\lib.win-amd64-3.6\rpy2\ipython
  creating build\lib.win-amd64-3.6\rpy2\ipython\tests
  copying .\rpy\ipython\tests\test_rmagic.py -> build\lib.win-amd64-3.6\rpy2\ipython\tests
  copying .\rpy\ipython\tests\__init__.py -> build\lib.win-amd64-3.6\rpy2\ipython\tests
  running build_clib
  building 'r_utils' library
  error: [WinError 3] The system cannot find the path specified: 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\PlatformSDK\\lib'
  
  ----------------------------------------
  Running setup.py clean for rpy2
Failed to build rpy2
Installing collected packages: rpy2
  Running setup.py install for rpy2: started
    Running setup.py install for rpy2: finished with status 'error'
    Complete output from command C:\Users\SHADRACK\Anaconda3\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\SHADRACK\\AppData\\Local\\Temp\\pip-build-ohwpm7w_\\rpy2\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\SHADRACK\AppData\Local\Temp\pip-uxymgzzq-record\install-record.txt --single-version-externally-managed --compile:
    R version 3.4.2 (2017-09-28) -- "Short Summer"
    Copyright (C) 2017 The R Foundation for Statistical Computing
    Platform: x86_64-w64-mingw32/x64 (64-bit)
    
    R is free software and comes with ABSOLUTELY NO WARRANTY.
    You are welcome to redistribute it under the terms of the
    GNU General Public License versions 2 or 3.
    For more information about these matters see
    http://www.gnu.org/licenses/.
    
    R version 3.4.2 (2017-09-28) -- "Short Summer"
    C:\Users\SHADRACK\Anaconda3\R\bin\x64\R CMD config --ldflags
    bash.exe: warning: could not find /tmp, please create!
    C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 177: make: command not found
    C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 178: make: command not found
    C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 179: make: command not found
    C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 180: make: command not found
    C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 181: make: command not found
    R was not built as a library
    C:\Users\SHADRACK\Anaconda3\R\bin\x64\R CMD config --cppflags
    bash.exe: warning: could not find /tmp, please create!
    C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 177: make: command not found
    C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 178: make: command not found
    C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 179: make: command not found
    C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 180: make: command not found
    C:\Users\SHADRACK\Anaconda3\R/bin/config.sh: line 181: make: command not found
    R was not built as a library
    C:\Users\SHADRACK\AppData\Local\Temp\pip-build-ohwpm7w_\rpy2\setup.py:282: UserWarning: No include specified
      warnings.warn('No include specified')
    C:\Users\SHADRACK\AppData\Local\Temp\pip-build-ohwpm7w_\rpy2\setup.py:293: UserWarning: No libraries as -l arguments to the compiler.
      warnings.warn('No libraries as -l arguments to the compiler.')
    
        Compilation parameters for rpy2's C components:
            include_dirs    = []
            library_dirs    = []
            libraries       = []
            extra_link_args = ['-m64']
    
    running install
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-3.6
    creating build\lib.win-amd64-3.6\rpy2
    copying .\rpy\rpy_classic.py -> build\lib.win-amd64-3.6\rpy2
    copying .\rpy\situation.py -> build\lib.win-amd64-3.6\rpy2
    copying .\rpy\tests.py -> build\lib.win-amd64-3.6\rpy2
    copying .\rpy\tests_rpy_classic.py -> build\lib.win-amd64-3.6\rpy2
    copying .\rpy\__init__.py -> build\lib.win-amd64-3.6\rpy2
    creating build\lib.win-amd64-3.6\rpy2\rlike
    copying .\rpy\rlike\container.py -> build\lib.win-amd64-3.6\rpy2\rlike
    copying .\rpy\rlike\functional.py -> build\lib.win-amd64-3.6\rpy2\rlike
    copying .\rpy\rlike\indexing.py -> build\lib.win-amd64-3.6\rpy2\rlike
    copying .\rpy\rlike\__init__.py -> build\lib.win-amd64-3.6\rpy2\rlike
    creating build\lib.win-amd64-3.6\rpy2\rlike\tests
    copying .\rpy\rlike\tests\test_container.py -> build\lib.win-amd64-3.6\rpy2\rlike\tests
    copying .\rpy\rlike\tests\test_functional.py -> build\lib.win-amd64-3.6\rpy2\rlike\tests
    copying .\rpy\rlike\tests\test_indexing.py -> build\lib.win-amd64-3.6\rpy2\rlike\tests
    copying .\rpy\rlike\tests\__init__.py -> build\lib.win-amd64-3.6\rpy2\rlike\tests
    creating build\lib.win-amd64-3.6\rpy2\rinterface
    copying .\rpy\rinterface\__init__.py -> build\lib.win-amd64-3.6\rpy2\rinterface
    creating build\lib.win-amd64-3.6\rpy2\rinterface\tests
    copying .\rpy\rinterface\tests\test_Device.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
    copying .\rpy\rinterface\tests\test_EmbeddedR.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
    copying .\rpy\rinterface\tests\test_Sexp.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
    copying .\rpy\rinterface\tests\test_SexpClosure.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
    copying .\rpy\rinterface\tests\test_SexpEnvironment.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
    copying .\rpy\rinterface\tests\test_SexpExtPtr.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
    copying .\rpy\rinterface\tests\test_SexpSymbol.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
    copying .\rpy\rinterface\tests\test_SexpVector.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
    copying .\rpy\rinterface\tests\test_SexpVectorNumeric.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
    copying .\rpy\rinterface\tests\__init__.py -> build\lib.win-amd64-3.6\rpy2\rinterface\tests
    creating build\lib.win-amd64-3.6\rpy2\robjects
    copying .\rpy\robjects\constants.py -> build\lib.win-amd64-3.6\rpy2\robjects
    copying .\rpy\robjects\conversion.py -> build\lib.win-amd64-3.6\rpy2\robjects
    copying .\rpy\robjects\environments.py -> build\lib.win-amd64-3.6\rpy2\robjects
    copying .\rpy\robjects\functions.py -> build\lib.win-amd64-3.6\rpy2\robjects
    copying .\rpy\robjects\help.py -> build\lib.win-amd64-3.6\rpy2\robjects
    copying .\rpy\robjects\language.py -> build\lib.win-amd64-3.6\rpy2\robjects
    copying .\rpy\robjects\methods.py -> build\lib.win-amd64-3.6\rpy2\robjects
    copying .\rpy\robjects\numpy2ri.py -> build\lib.win-amd64-3.6\rpy2\robjects
    copying .\rpy\robjects\packages.py -> build\lib.win-amd64-3.6\rpy2\robjects
    copying .\rpy\robjects\packages_utils.py -> build\lib.win-amd64-3.6\rpy2\robjects
    copying .\rpy\robjects\pandas2ri.py -> build\lib.win-amd64-3.6\rpy2\robjects
    copying .\rpy\robjects\robject.py -> build\lib.win-amd64-3.6\rpy2\robjects
    copying .\rpy\robjects\vectors.py -> build\lib.win-amd64-3.6\rpy2\robjects
    copying .\rpy\robjects\__init__.py -> build\lib.win-amd64-3.6\rpy2\robjects
    creating build\lib.win-amd64-3.6\rpy2\robjects\tests
    copying .\rpy\robjects\tests\testArray.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
    copying .\rpy\robjects\tests\testDataFrame.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
    copying .\rpy\robjects\tests\testEnvironment.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
    copying .\rpy\robjects\tests\testFormula.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
    copying .\rpy\robjects\tests\testFunction.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
    copying .\rpy\robjects\tests\testHelp.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
    copying .\rpy\robjects\tests\testLanguage.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
    copying .\rpy\robjects\tests\testMethods.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
    copying .\rpy\robjects\tests\testNumpyConversions.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
    copying .\rpy\robjects\tests\testPackages.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
    copying .\rpy\robjects\tests\testPandasConversions.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
    copying .\rpy\robjects\tests\testRObject.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
    copying .\rpy\robjects\tests\testRobjects.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
    copying .\rpy\robjects\tests\testVector.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
    copying .\rpy\robjects\tests\__init__.py -> build\lib.win-amd64-3.6\rpy2\robjects\tests
    creating build\lib.win-amd64-3.6\rpy2\robjects\lib
    copying .\rpy\robjects\lib\dplyr.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib
    copying .\rpy\robjects\lib\ggplot2.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib
    copying .\rpy\robjects\lib\grdevices.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib
    copying .\rpy\robjects\lib\grid.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib
    copying .\rpy\robjects\lib\test_dplyr.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib
    copying .\rpy\robjects\lib\tidyr.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib
    copying .\rpy\robjects\lib\__init__.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib
    creating build\lib.win-amd64-3.6\rpy2\robjects\lib\tests
    copying .\rpy\robjects\lib\tests\test_dplyr.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib\tests
    copying .\rpy\robjects\lib\tests\test_ggplot2.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib\tests
    copying .\rpy\robjects\lib\tests\test_grdevices.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib\tests
    copying .\rpy\robjects\lib\tests\__init__.py -> build\lib.win-amd64-3.6\rpy2\robjects\lib\tests
    creating build\lib.win-amd64-3.6\rpy2\interactive
    copying .\rpy\interactive\packages.py -> build\lib.win-amd64-3.6\rpy2\interactive
    copying .\rpy\interactive\process_revents.py -> build\lib.win-amd64-3.6\rpy2\interactive
    copying .\rpy\interactive\__init__.py -> build\lib.win-amd64-3.6\rpy2\interactive
    creating build\lib.win-amd64-3.6\rpy2\interactive\tests
    copying .\rpy\interactive\tests\__init__.py -> build\lib.win-amd64-3.6\rpy2\interactive\tests
    creating build\lib.win-amd64-3.6\rpy2\ipython
    copying .\rpy\ipython\ggplot.py -> build\lib.win-amd64-3.6\rpy2\ipython
    copying .\rpy\ipython\html.py -> build\lib.win-amd64-3.6\rpy2\ipython
    copying .\rpy\ipython\rmagic.py -> build\lib.win-amd64-3.6\rpy2\ipython
    copying .\rpy\ipython\__init__.py -> build\lib.win-amd64-3.6\rpy2\ipython
    creating build\lib.win-amd64-3.6\rpy2\ipython\tests
    copying .\rpy\ipython\tests\test_rmagic.py -> build\lib.win-amd64-3.6\rpy2\ipython\tests
    copying .\rpy\ipython\tests\__init__.py -> build\lib.win-amd64-3.6\rpy2\ipython\tests
    running build_clib
    building 'r_utils' library
    error: [WinError 3] The system cannot find the path specified: 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\PlatformSDK\\lib'
    
    ----------------------------------------
  Failed building wheel for rpy2
Command "C:\Users\SHADRACK\Anaconda3\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\SHADRACK\\AppData\\Local\\Temp\\pip-build-ohwpm7w_\\rpy2\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\SHADRACK\AppData\Local\Temp\pip-uxymgzzq-record\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in C:\Users\SHADRACK\AppData\Local\Temp\pip-build-ohwpm7w_\rpy2\`

**I am using spyder provided under anaconda distribution**

## Versions and main components
 sys.version
*Out[47]: '3.6.3 |Anaconda, Inc.| (default, Oct 15 2017, 03:27:45) [MSC v.1900 64 bit (AMD64)]*
* Spyder(python 3.6):
* Python Version 3.6:
* Windows 10:

## Dependencies

Please go to the menu entry `Help > Optional Dependencies` (or
`Help > Dependencies`), press the button `Copy to clipboard`
and paste the contents below:

IPython >=4.0    :  6.1.0 (OK)
cython >=0.21    :  0.26.1 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
numpy >=1.7      :  1.13.3 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pyflakes >=0.6.0 :  1.6.0 (OK)
pygments >=2.0   :  2.2.0 (OK)
pylint >=0.25    :  1.7.4 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
rope >=0.9.4     :  0.10.5 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
sympy >=0.7.3    :  1.1.1 (OK)

Please help.

**Comment by CAM-Gerlach** [comment]
```
error: [WinError 3] The system cannot find the path specified: 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\PlatformSDK\\lib'
```
Perhaps something is missing with your VC compiler? No idea.

In any case, I'm sorry, but I'm really not sure how this is any way Spyder-related, or even directly Python related as building R seems to be the problem. Perhaps you would have more luck posting an issue on the rpy issue tracker, assuming you've Googled it and the problem doesn't appear to be on your end. Best of luck!

**Comment by ccordoba12** [comment]
> Perhaps something is missing with your VC compiler?

Yes, that's right @CAM-Gerlach!

----

@shadrackkibet, two things are very important here:

1. Please don't use `!pip install foo` or `!conda install foo` inside our IPython consoles. Those commands are meant to be run in a proper terminal (i.e. `cmd.exe`).
2. Please avoid using `pip` if there's a conda package available. So instead of running `pip install rpy2` (which requires a C/C++ compiler), please run instead `conda install rpy2` or `conda install -c conda-forge rpy2`, which doesn't need one.

To recap, the solution to your problem is to run in a terminal

     conda install -c conda-forge rpy2
